ephemeral/startup/action.yml

```diff
@@ -102,9 +102,9 @@ runs:
         echo "AWS_ENDPOINT_URL=$endpointUrl" >> $GITHUB_ENV
 
     - name: Upload preview instance URL
-      uses: actions/upload-artifact@v3
+      uses: actions/upload-artifact@v4
       with:
-        name: pr
+        name: preview-instance-url
```

Comment:
**suggestion**: rename to `pr-id` to keep existing functionality working

Reply:
**follow-up thought**: keeping as `preview-instance-url`, since we cannot upload two artifacts with the same name, we can download this _additionally_ wherever it's needed.